monitoring/status-page/status/routes.py

```diff
@@ -0,0 +1,23 @@
+import argparse
+import json
+import requests
+from flask import render_template
+from status import app
+
+watchdog = 'http://watchdog.hmny.io/status-%s'
+network_list = [('Mainnet', 'mainnet'),('LRTN', 'testnet'), ('OSTN', 'staking')]
+
+@app.route('/status')
+def status():
+    statuses = {}
+    for n in network_list:
+        r = requests.get(watchdog % n[1], auth=('harmony', 'harmony.one'))
```

Comment:
no need to do the authentication, it should be public.

Reply:
Authentication is for getting the data from existing Watchdog services.